pkg/chunk/schema_config.go

```diff
@@ -295,6 +295,39 @@ type PeriodicTableConfig struct {
 	Tags   Tags          `yaml:"tags,omitempty"`
```

Comment:
I think you can remove all `yaml` tags here because you have a custom marshalling. If I'm not missing anything, leaving them here could lead to misunderstanding to future readers.